Lib/_strptime.py

```diff
@@ -202,7 +202,10 @@ def __init__(self, locale_time=None):
             #XXX: Does 'Y' need to worry about having less or more than
             #     4 digits?
             'Y': r"(?P<Y>\d\d\d\d)",
+            # "z" shouldn't support colons. Both ":?" should be removed. However, for backwards
```

Comment:
```suggestion
            # "z" might not support colons, if designed from scratch. However, for backwards
```

Again: the statement that it *shoudn't* support colons is too strong IMHO. It is perfectly OK, that it supports them.